### ISSUE TITLE
After Sign in on IDM, End-User not automatically redirected to portal

### ISSUE THREAD

**Issue by prudhomm** [MSO4SC/MSOPortal#89]
I am stuck in IDM after sign in (http://portal.mso4sc.eu:3000/idm)  I am not automatically redirected to the portal once I sign in.
The Expected Behavior is after sign in, I am redirected to the portal.
This is not good for User Experience.

**Comment by prudhomm** [comment]
@VedatDurmaz I believe you had the same issue, right ?

**Comment by VedatDurmaz** [comment]
@prudhomm yes, i observe exactly the same behaviour. but, to be honest, i actually don't know what is the expected behaviour (and intention!) of that site. shame over me, i know  ;)

**Comment by emepetres** [comment]
I think this could be closely related to #88.
@prudhomm @VedatDurmaz could you perform the tests suggested on #88 for this issue as well?

**Comment by lsala** [comment]
I have the same problem, especially when I created the account then it was impossible to find the link to the portal

**Comment by emepetres** [comment]
Please note that httpS://portal.mso4sc.eu is IDM and http://portal.mso4sc.eu is Portal.
